src/test/java/de/poiu/apron/io/PropertyFileReaderTest.java

```diff
@@ -425,6 +426,67 @@ public void testSingleBackslashOnLine() throws IOException {
     assertThat(readEntries.get(1)).isEqualTo(new PropertyEntry("   ", "\\ ", "   ", "", "\n"));
   }
 
+  /**
+   * This test verifies bug #11: https://github.com/hupfdule/apron/issues/11
+   * @throws IOException
+   */
+  @Test
+  public void test_CrLfEndings() throws IOException {
+    // - preparation
+    final File propertyFile= this.createTestFile(""
+        + "keyA1=valueA1\n"
+        + " keyA2  =  valueA2\n"
+        + "\tkeyA3\t=\tvalue A3\t\n"
+        + "keyA4 = very long\\\r\n"
+        + "value A4 over \\\r\n"
+        + "multiple lines\r\n"
+        + "        \n"
+        + "keyB1:valueB1\n"
+        + " keyB2 : valueB2\n"
+        + "\t keyB3\t:\t value B3 \n"
+        + "keyB4 : very long\\\n"
+        + "value B4 over \\\n"
+        + "multiple lines\\\n"
+        + "\n"
+        + "keyC1 valueC1\n"
+        + "  keyC2   valueC2\n"
+        + "\t keyC3\t\tvalue C3 \n"
+        + "keyC4   very long\\\n"
+        + "value C4 over \\\n"
+        + "\t \tmultiple lines");
```

Comment:
These are way too many unrelated tests. It should be reduced to the actual problematic ones. I would therefore propose the following test data:

```suggestion
      + "lf                       = One\\\nTwo\n"
      + "cr                       = One\\\rTwo\r"
      + "crlf_both_escaped        = One\\\r\\\nTwo\r\n"
      + "crlf_only_first_escaped  = One\\\r\nTwo\r\n");
```

This tests the exact same value with different line termination characters. All should result in the very same value to be parsed.